de-de/scala-de.html.markdown

```diff
@@ -0,0 +1,810 @@
+vorkommen und man auch mit einer Lib die Implicits benutzt nichts sinnvolles  
+machen kann.  
+Hier soll ein Grundverständnis geschaffen werden, wie sie funktionieren.  
+
+// Mit dem Schlüsselwort implicit können Methoden, Werte, Funktion, Objekte  
+// zu "implicit Methods" werden.  
+```
+implicit val myImplicitInt = 100
+implicit def myImplicitFunction(sorte: String) = new Hund("Golden " + sorte)
+```
+
+// implicit ändert nicht das Verhalten eines Wertes oder einer Funktion  
+```
+myImplicitInt + 2                   // => 102
+myImplicitFunction("Pitbull").sorte // => "Golden Pitbull"
+```
+
+// Der Unterschied ist, dass diese Werte ausgewählt werden können, wenn ein  
+// anderer Codeteil einen implicit Wert benötigt, zum Beispiel innerhalb von  
+// implicit Funktionsparametern  
+  
+// Diese Funktion hat zwei Parameter: einen normalen und einen implicit  
+```
+def sendGreetings(toWhom: String)(implicit howMany: Int) =
+  s"Hello $toWhom, $howMany blessings to you and yours!"
+```
+
+// Werden beide Parameter gefüllt, verhält sich die Funktion wie erwartet  
+```
+sendGreetings("John")(1000)  // => "Hello John, 1000 blessings to you and yours!"
+```
+
+// Wird der implicit Parameter jedoch weggelassen, wird ein anderer  
+// implicit Wert vom gleichen Typ genommen. Der Compiler sucht im  
+// lexikalischen Scope und im companion object nach einem implicit Wert,  
+// der vom Typ passt, oder  nach einer implicit Methode mit der er in den  
+// geforderten Typ konvertieren kann.  
+  
+// Hier also: "myImplicitInt", da ein Int gesucht wird  
+```
+sendGreetings("Jane")  // => "Hello Jane, 100 blessings to you and yours!"
+```
+
+// bzw. "myImplicitFunction"  
+// Der String wird erst mit Hilfe der Funktion in Hund konvertiert, und  
+// dann wird die Methode aufgerufen  
+```
+"Retriever".sorte // => "Golden Retriever"
+```
+
+# 9. Misc
+## Importe
+```
+import scala.collection.immutable.List
+```
+
+// Importiere alle Unterpackages  
+```
+import scala.collection.immutable._
+```
+
+// Importiere verschiedene Klassen mit einem Statement  
+```
+import scala.collection.immutable.{List, Map}
+```
+
+// Einen Import kann man mit '=>' umbenennen  
+```
+import scala.collection.immutable.{List => ImmutableList}
+```
+
+// Importiere alle Klasses, mit Ausnahem von....  
+// Hier ohne: Map and Set:  
+```
+import scala.collection.immutable.{Map => _, Set => _, _}
+```
+## Main 
+```
+object Application {
+  def main(args: Array[String]): Unit = {
+    // stuff goes here.
+  }
+}
+```
+
+## I/O
+// Eine Datei Zeile für Zeile lesen  
+```
+import scala.io.Source
+for(line <- Source.fromFile("myfile.txt").getLines())
+  println(line)
+```
+
+// Eine Datei schreiben  
+```
+val writer = new PrintWriter("myfile.txt")
+writer.write("Writing line for line" + util.Properties.lineSeparator)
+writer.write("Another line here" + util.Properties.lineSeparator)
+writer.close()
+```
+
+## Further resources
```

Comment:
I've added german resources, translated the heading and marked EN resources as such.